docker/docker-compose.vs.yml

```diff
@@ -0,0 +1,20 @@
+version: "3.4"
+
+services:
+  microsoft.health.dicom.web:
+    image: ${DOCKER_REGISTRY-}microsofthealthdicomweb
+    environment:
+      ASPNETCORE_ENVIRONMENT: "Development"
+      ASPNETCORE_URLS: "https://+:8080"
+      DicomServer__Security__Enabled: "false"
+      # [SuppressMessage("Microsoft.Security", "CS002:SecretInNextLine", Justification="Password is only used for development.")]
+      SqlServer__ConnectionString: "Server=tcp:sql,1433;Initial Catalog=DICOM;Persist Security Info=False;User ID=sa;Password=T3stP@ssw0rd;MultipleActiveResultSets=False;Connection Timeout=30;"
+    user: root
+    ports:
+      - "63839:8080"
+    volumes:
+      - ${APPDATA}/ASP.NET/Https:/root/.aspnet/https:ro
+  sql:
+    environment:
+      # [SuppressMessage("Microsoft.Security", "CS002:SecretInNextLine", Justification="Password is only used for development.")]
+      SA_PASSWORD: T3stP@ssw0rd
```

Comment:
>T3stP@ssw0rd [](http://example.com/codeflow?start=19&length=12)

may want default values for these
[https://github.com/compose-spec/compose-spec/blob/master/spec.md#interpolation](https://github.com/compose-spec/compose-spec/blob/master/spec.md#interpolation)

Reply:
Makes sense!